src/SprykerEco/Zed/Unzer/Business/Payment/Finder/UnzerMarketplacePaymentUnzerCredentialsFinder.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Payment\Finder;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\UnzerCredentialsConditionsTransfer;
+use Generated\Shared\Transfer\UnzerCredentialsCriteriaTransfer;
+use Generated\Shared\Transfer\UnzerCredentialsTransfer;
+use Generated\Shared\Transfer\UnzerMarketplacePaymentCredentialsFinderCriteriaTransfer;
+use SprykerEco\Shared\Unzer\UnzerConstants;
+use SprykerEco\Zed\Unzer\Business\ApiAdapter\UnzerPaymentMethodsAdapterInterface;
+use SprykerEco\Zed\Unzer\Business\Exception\UnzerException;
+use SprykerEco\Zed\Unzer\Business\Reader\UnzerReaderInterface;
+
+class UnzerMarketplacePaymentUnzerCredentialsFinder implements UnzerMarketplacePaymentUnzerCredentialsFinderInterface
```

Comment:
I would say that `Resolver` name better suits this class

Reply:
Fixed.